script/clone_job.pl

```diff
@@ -111,10 +111,8 @@ =head1 SYNOPSIS
 Getopt::Long::Configure("no_ignore_case");
 use Mojo::URL;
 use JSON;
-
 use FindBin;
-use lib "$FindBin::Bin/../lib";
-use OpenQA::Client;
```

Comment:
I guess this is the culprit.

Reply:
yes, pretty sure it is. Great you catched it, fixed in https://github.com/os-autoinst/openQA/pull/1463